tfhe/src/core_crypto/commons/math/random/gaussian.rs

```diff
@@ -74,6 +74,15 @@ macro_rules! implement_gaussian {
                 // probability
                 std::f64::consts::PI / 4.0
             }
+
+            fn single_sample_required_random_byte_count(
+                _distribution: Gaussian<$T>,
+                _modulus: Option<Self::CustomModulus>,
+            ) -> usize {
+                // The modulus and parameters of the distribution do not impact the amount of byte
+                // required
+                2 * std::mem::size_of::<$S>()
```

Comment:
Why `2*`?

Reply:
we fill two buffers see line 46 and following